library/src/main/java/com/bumptech/glide/signature/EmptySignature.java

```diff
@@ -9,6 +10,7 @@
 public final class EmptySignature implements Key {
   private static final EmptySignature EMPTY_KEY = new EmptySignature();
```

Comment:
Yeah, `EMPTY_KEY` is @NonNull too

Reply:
@anticafe thanks for pointing this out, mind sending a pull request to update those too? Thanks!